tests/test_document.py

```diff
@@ -755,7 +755,57 @@ def test_issue_url(self):
 
     def test_collection_acronym(self):
 
-        self.assertTrue(self.issue.url, '')
+        self.assertTrue(self.issue.collection_acronym, '')
+
+    def test_issue_number_label(self):
+        self.issue.data['issue']['v32'] = [{u'_': u'2'}]
+        self.assertEqual(self.issue.number_label, u'2')
+        self.issue.data['issue']['v32'] = [{u'_': u'0'}]
+        self.assertEqual(self.issue.number_label, u'')
+        self.issue.data['issue']['v32'] = [{u'_': u'5'}]
+        self.assertEqual(self.issue.number_label, u'5')
+        del self.issue.data['issue']['v32']
+        self.assertEqual(self.issue.number_label, u'')
+
+    def test_special_number_label(self):
+        self.issue.data['issue']['v32'] = [{u'_': u'spe'}]
+        self.assertEqual(self.issue.number_label, u'')
+        self.issue.data['issue']['v32'] = [{u'_': u'spe2'}]
+        self.assertEqual(self.issue.number_label, u'')
+        self.issue.data['issue']['v32'] = [{u'_': u'5spe'}]
+        self.assertEqual(self.issue.number_label, u'5')
+        self.issue.data['issue']['v32'] = [{u'_': u'5spe3'}]
+        self.assertEqual(self.issue.number_label, u'5')
+
+    def test_supplement_number_label(self):
```

Comment:
@patymori Ou eu não entendi a função do number_label ou isso não está correto.

v31 = volume do fascículo (poderia ser alfanumérico ou inexistente)
v32 = número do fascículo. Pode ser alfanumérico, por exemplo 1-2, 3B, ou inexistente ou ainda ter indicação de número especial se contiver como convenção spe. 
v131 = suplemento do volume. É alfanumérico, mas se for 0, a tradução é Suplemento ou Supplement), se for 1, a tradução é Suplemento 1 ou Supplement 1). Existindo o v131, significa que v32 é ausente.
v132 = suplemento de número. É similar ao v131. A diferença é que havendo v132, obrigatoriamente há o v32.

Sendo assim, imaginava que o number_label somente fosse aplicável ao campo v32 e não também ao campo v132.
Se há v32  e v132, porque o retorno tem que ser o número encontrado em v132? O valor em v32 é descartado?